ci/etc/configure_node.sh

```diff
@@ -7,7 +7,8 @@ fi
 if [ "x${NODE_TYPE}" == "xmom" ]; then
 	sed -i "s@PBS_SERVER=$(hostname)@PBS_SERVER=pbs_server@" /etc/pbs.conf
 	sed -i "s@PBS_START_SERVER=1@PBS_START_SERVER=0@" /etc/pbs.conf
-	HOST_NAME=$(hostname -f)
+	HOST_NAME=$(hostname -s)
+	HOST_NAME="${HOST_NAME}.ci_ci"
 	ssh -t root@pbs_server " /opt/pbs/bin/qmgr -c 'c n ${HOST_NAME}'"
```

Comment:
I think we should create a node with the short name instead fqdn, also just a suggestion make fqdn like `name.awesome.ci` (aka more that one dot and without `_`)

Reply:
Using the default DNS for the container names, which docker-compose assigns as {folder_name}_{network name}, hence it will be ci_ci. While i was trying the custom domainname, although the nslookup was working for (hostname -s) but not for FQDN name. So i used the default DNS which avoids complications,and ssh works well.

Reply:
we can override it, I am using custom names in my machine like `node1.dev.local`, `node2.dev.local` etc.. and everything works fine.

let's discuss offline, and will find solution for it.